waisntechtools/alerts/tests/test_messenger.py

```diff
@@ -0,0 +1,15 @@
+import tempfile
+
+from django.test import TestCase
+
+from alerts.messenger import Messenger, Message
+from alerts.models import Subscriber
+
+
+class MessageTestCase(TestCase):
+    def test_message(self):
+        temp = tempfile.NamedTemporaryFile()
```

Comment:
it would be more pythonic to have use the resource manager syntax so the resource is cleaned up before the test ends.

```
with tempfile.NamedTemporaryFile() as temp:
        temp.write('The test has completed'.encode(encoding='utf-8'))
        temp.flush()
        read_message = Message(temp.name).contents()
        self.assertEqual(read_message, 'The test has completed')
```